karate/src/test/java/api/entities/entities-create.feature

```diff
@@ -0,0 +1,58 @@
+@entities
+Feature: Test implementation of POST /entities (6.4.3.1)
+
+Background:
+  * url urlBase
+  * def buildings = read('classpath:samples/buildings.json')
+  * def buildingMinimal = read('classpath:samples/building-minimal.jsonld')
+  * def building = read('classpath:samples/building.jsonld')
+  * def buildingWithoutContext = read('classpath:samples/building-without-context.json')
+  * configure charset = null
```

Comment:
Is it needed?

Reply:
Yes, otherwise the `Create Building with wrong Media type (application/json with JSON-LD payload)` scenario fails.